deployments/helm/signalfx-k8s-metrics-adapter/values.yaml

```diff
@@ -9,3 +9,10 @@ imageTag: 0.2.0
 imagePullPolicy: IfNotPresent
 args:
 
+resources:
```

Comment:
It is better to not provide any defaults for resources and just let it continue without resource bounds by default.

Reply:
@keitwb comment addressed, please help check, thanks.